src/quo2/components/info/page_nav.cljs

```diff
@@ -0,0 +1,242 @@
+(ns quo2.components.info.page-nav
+  (:require [quo.react-native :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as icons]
+            [status-im.utils.dimensions :as dimensions]
+            [clojure.string :as string]
+            [quo2.components.markdown.text :as text]))
+
+(def ^:private centrify
+  {:style {:display :flex
+           :justify-content :center
+           :align-items     :center}})
+
+(def ^:private align-left
+  (update-in
+   centrify
+   [:style]
+   #(assoc
+     %
+     :align-items :flex-start)))
+
+(def ^:private icon-styles
+  (update-in
+   centrify
+   [:style]
+   #(assoc
+     %
+     :width         32
+     :height        32
+     :border-radius 10)))
+
+(defn- big?
+  [size]
+  (= size :big))
+
+(defn- icon-props
+  [color size]
+  (merge {:container-style
+          {:width  (if (big? size)
+                     20
+                     16)
+           :height (if (big? size)
+                     20
+                     16)}
+          :size      20}
+         (if-not (string/blank? color)
+           {:color    color}
+           {:no-color true})))
+
+(defn- mid-section-comp
+  [{:keys [mid-section-description-user-icon
+           horizontal-description?
+           text-secondary-color
+           align-mid?
+           text-color
+           mid-section-icon
+           mid-section-main-text
+           mid-section-type
+           mid-section-description]}]
+  [rn/view (update-in
+            centrify
+            [:style]
+            #(assoc % :flex-direction    :row
+                    :margin-horizontal 2))
+   (when (or (and (not horizontal-description?)
+                  align-mid?
+                  (not (= :text-with-description mid-section-type)))
+             (and mid-section-description-user-icon
+                  (not mid-section-icon)))
+     [rn/image {:source {:uri mid-section-description-user-icon}
+                :style  {:width         32
+                         :height        32
+                         :border-radius 32
+                         :margin-right  8}}])
+   [rn/view {:style {:flex-direction (if horizontal-description?
+                                       :row
+                                       :column)}}
+    [text/text {:size   :paragraph-1
+                :weight :semi-bold
+                :style  {:color       text-color
+                         :line-height 21}}
+     mid-section-main-text]
+    (when mid-section-description
+      [text/text {:size   :paragraph-2
+                  :weight :medium
+                  :style  (cond-> {:padding-right 4
+                                   :color         text-secondary-color
+                                   :line-height   18}
+                            horizontal-description? (assoc :margin-left 4 :margin-top 2))}
+
+       mid-section-description])]])
+
+(defn- mid-section
+  [{:keys [horizontal-description?
```

Comment:
what's the point of having each key on the new line ? i mean most of the time you don't need them like that, its hard to read code when you have so many lines

Reply:
A previous senior developer told me to keep it that way, I'll revert it to be multiple arguments on each line